From b9cc1b4832f4ebdc644a5aa2cf02e4ac40c56451 Mon Sep 17 00:00:00 2001
From: Ralph Bean <rbean@redhat.com>
Date: Tue, 28 Jan 2014 16:18:17 -0500
Subject: [PATCH] Disable epgm test

---
 txzmq/test/test_pubsub.py | 26 --------------------------
 1 file changed, 26 deletions(-)

diff --git a/txzmq/test/test_pubsub.py b/txzmq/test/test_pubsub.py
index 86a0238..f2fb12e 100644
--- a/txzmq/test/test_pubsub.py
+++ b/txzmq/test/test_pubsub.py
@@ -19,21 +19,6 @@ class ZmqTestSubConnection(ZmqSubConnection):
         self.messages.append([tag, message])
 
 
-def _detect_epgm():
-    """
-    Utility function to test for presence of epgm:// in zeromq.
-    """
-    import zmq
-
-    context = zmq.Context()
-    socket = zmq.Socket(context, zmq.constants.PUB)
-
-    try:
-        socket.bind("epgm://127.0.0.1;239.192.1.1:5557")
-
-        return True
-    except ZMQError:
-        return False
 
 
 class ZmqConnectionTestCase(unittest.TestCase):
@@ -81,14 +66,6 @@ class ZmqConnectionTestCase(unittest.TestCase):
         return _wait(0.01).addCallback(publish) \
             .addCallback(lambda _: _wait(0.01)).addCallback(check)
 
-    def test_send_recv_pgm(self):
-        r = ZmqTestSubConnection(self.factory, ZmqEndpoint(
-            ZmqEndpointType.bind, "epgm://127.0.0.1;239.192.1.1:5556"))
-
-        s = ZmqPubConnection(self.factory, ZmqEndpoint(
-            ZmqEndpointType.connect, "epgm://127.0.0.1;239.192.1.1:5556"))
-
-        r.subscribe('tag')
 
         def publish(ignore):
             s.publish('xyz', 'different-tag')
@@ -130,6 +107,3 @@ class ZmqConnectionTestCase(unittest.TestCase):
 
         return _wait(0.1).addCallback(publish) \
             .addCallback(lambda _: _wait(0.1)).addCallback(check)
-
-    if not _detect_epgm():
-        test_send_recv_pgm.skip = "epgm:// not available"
-- 
1.8.5.3

